pyaedt/edb_core/edb_data/sources.py

```diff
@@ -505,3 +505,20 @@ def pec_launch_width(self):
         """Get pec launch width."""
         txt = re.search(r"'PEC Launch Width'='.*?'", self._edb_properties).group()
         return txt.split("=")[1].replace("'", "")
+
+
+class ExcitationDifferential:
+    def __init__(self, pedb, edb_boundle_terminal, name):
+        self._pedb = pedb
+        self._edb_boundle_terminal = edb_boundle_terminal
+        self._name = name
+
+    @property
+    def _edb(self):
```

Comment:
@ring630 Why not naming the property `edb` instead?

Reply:
Done